engines/router/missionctl/instrumentation/metrics/metrics.go

```diff
@@ -80,3 +80,18 @@ func GetStatusString(status bool) string {
 	}
 	return statusLabels.Failure
 }
+
+// GetMeasureDurationFunc return the func that measures the duration of the request
+func GetMeasureDurationFunc(err error, componentID string) func() {
```

Comment:
This function is hardcoding the metric name (TuringComponentRequestDurationMs) and labels. So I feel it doesn't belong in this package where the functions are generic. Also based on the comment on `engines/router/missionctl/fiberapi/fan_in.go`, I'm not convinced we need it.